src/components/TravelInfo.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import { TextField, InputLabel, Typography } from "@mui/material";
+import style from "../components/style//TravelInfo.module.css";
+
+function TravelInfo(props: any) {
+  let travelInfoData = {};
+
+  const [form, setForm] = useState({});
+
+  function handleForm(e: React.ChangeEvent<HTMLInputElement>, key: string) {
+    setForm({
+      ...form,
+      [key]: e.target.value,
+    });
+  }
+  travelInfoData = {
```

Comment:
Do you need this as a separate variable? You can use the value from state directly.

Reply:
I fix it.